modules/testkit/shared/src/main/scala-3/cron4s/testkit/gen/CronGenerators.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 Antonio Alonso Dominguez
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cron4s.testkit.gen
+
+import cron4s.CronField
+import cron4s.expr._
+
+import org.scalacheck._
+
```

Comment:
This file is almost the same as https://github.com/alonsodomin/cron4s/pull/504/files#diff-fe082e93829b26c97f89813516bdfd1efa4b2f6cc1d26d5dcd7aa2dcec929e03.

The differences are
- explicit type annotations for implicit vals/defs
- shapeless import removal